server/service/src/print/label.rs

```diff
@@ -79,28 +79,28 @@ pub fn print_prescription_label(
                 ^CI28
 
                 ^A0,25
-                ^FO10,10
-                ^FB570,2,0,C
-                ^FD{item_details}^FS
+                ^FO5,10
+                ^FB550,2,0,C
+                ^FD{item_details}\&^FS
 
-                ^FO10,65
+                ^FX Line
+                ^FO5,65
                 ^GB570,2,2^FS
 
                 ^A0,25
-                ^FO10,75
-                ^FB570,6,0,L
-                ^FD{item_directions}\&{warning}^FS
+                ^FO5,75
+                ^FB550,5,0,L
+                ^FD{item_directions}\&{warning}\&^FS
 
-                ^FO10,210
+                ^FX Line
+                ^FO5,210
                 ^GB570,2,2^FS
 
-                ^A0,25
-                ^FO10,220
-                ^FD{patient_details}^FS
+                ^A0,20
+                ^FO5,220
+                ^FB550,3,0,C
+                ^FD{patient_details}\&{details}\&^FS
```

Comment:
Main change here is putting patient details and details in a text box with word wrap rather than explicit lines.